### PR TITLE
Support chaining on AudioNode.connect() and AudioParam automation methods

### PR DESCRIPTION
**Pull request by hoch**
- Revert: AudioNode.connect(AudioParam) does not return anything. (see #650)
- Also changed the weird location of 'parameters' in the method description.

### CODE REVIEW

**Comment by hoch** [comment]
I reverted back to the current HEAD and re-applied the changes without redundant edit. (Sorry that we lost our comments in there.)

File: index.html [hunk 1]
```diff
@@ -2388,9 +2389,13 @@ <h2 id="AudioParam">
               "#widl-AudioParam-exponentialRampToValueAtTime-void-float-value-double-endTime">
               exponentialRampToValueAtTime</a></code>, respectively.
             </p>
+            <p>
+              This method returns the current <a>AudioParam</a> object for easy
```
Comment (rtoy) [inline index.html]: I didn't find anything about AudioNode connect() allowing easy chaining in the spec (did I miss it?), so I don't see why we need to say anything about easy chaining.  I think it's fairly obvious you can chain them together now.

Comment (hoch) [inline index.html]: Then should I remove all the instances of this paragraph as whole?

Comment (rtoy) [inline index.html]: No, I think you need to say what the actual returned object is.  I would just remove the easy chaining part.

**Pull request merged**